modules/configuring-trustyai-with-a-database.adoc

```diff
@@ -5,7 +5,8 @@
 
 [role='_abstract']
 
-If you have a relational database in your {openshift-platform} cluster such as MySQL or MariaDB, you can configure TrustyAI to use your database instead of a persistent volume claim (PVC). Using a database instead of a PVC for storage can improve scalability, performance, and data management in TrustyAI.
+If you have a relational database in your {openshift-platform} cluster such as MySQL or MariaDB, you can configure TrustyAI to use your database instead of a persistent volume claim (PVC). Using a database instead of a PVC for storage can improve scalability, performance, and data management in TrustyAI. 
+TrustyAI needs to be provided a database configuration secret before deployment. You can create a secret or specify the name of an existing Kubernetes secret within your project. 
```

Comment:
Avoid passive voice. Reword to something like:
You must provide TrustyAI with a database configuration secret before deployment.